style.css

```diff
@@ -78,3 +78,250 @@ span {
 .media {
   padding: 10px;
 }
+
+#twitter {
+  margin-top: 15px;
+}
+
+.recent-works {
+  font-family: "inter", sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.scroll-down {
+  margin-left: 140px;
+  text-align: center;
+  border-width: 0.01em;
+  background: white;
+  border-style: solid;
+  border-color: rgb(211, 210, 210);
+  border-radius: 50%;
+}
+
+.indicator {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.container3,
+.container4,
+.container5,
+.container6,
+.container7,
+.container8 {
+  display: flex;
+  width: 290px;
+  margin-right: 0.5px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item3,
+.item4,
+.item5,
+.item6,
+.item7,
+.item8 {
+  margin-bottom: 30px;
+}
+
+.see-project {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-size: medium;
+  text-align: center;
+  color: white;
+  height: 60px;
+  width: 120px;
+  background: #36b37f;
+  border-radius: 4px;
+  border-style: none;
+}
+
+.see-project:hover {
+  transition: background-color 0.5s;
```

Comment:
- [ ] **OPTIONAL:** Please use :focus property to implement the button interactions, it is one of the important parts actually to make your portfolio awesome.